loculus_values/environment_specific_values/production.yaml

```diff
@@ -88,20 +88,20 @@ resources:
       memory: "10Gi"
   keycloak:
     requests:
-      memory: "500Mi"
+      memory: "3Gi"
```

Comment:
This seems a bit too much per https://www.keycloak.org/high-availability/concepts-memory-and-cpu-sizing#:~:text=Recommendations%3A,connects%20to%20only%20one%20client. but I may be misunderstanding that?

Reply:
We have plenty of RAM, doesn't harm to reserve it - but yeah we could reduce if you like. We're not RAM constrained right now though.

Reply:
I'll reduce it to 1.5GB